cards.js

```diff
@@ -13,11 +13,37 @@ function shuffle(array){
     array[currentIndex]  = array[randomIndex];
     array[randomIndex]   = temporaryValue;
   }
-  return array;
+  return(array);
 }
 
+
 function buildDeck(){
 
-// Put some code in here!
+for(var i = 0; i < suits.length; i++){
+  for(var x = 0; x < values.length; x++){
+    deck.push(suits[i] + ' ' + values[x]);
+    }
+  }
+    return deck
```

Comment:
There are a few places where coding style is inconsistent. I'll make notes where I see them, but they don't affect functionality or completeness of your code. For example, indentation on this line is inconsistent.